spec/PeriodicBackgroundSync-index.bs

```diff
@@ -64,21 +65,22 @@ The <code>periodicsync</code> event is considered to run <dfn>in the background<
 
 The user agent is considered to be [=online=] if the user agent has established a network connection. A user agent MAY use a stricter definition of being [=online=]. Such a stricter definition MAY take into account the particular [=service worker=] or origin a [=periodicsync registration=] is associated with.
 
-#TODO: link to PeriodicSyncManager.register().
-A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact interval of which is decided by the user agent.
+A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact interval of which is decided by the user agent. This can be requested through {{PeriodicSyncManager}}.
 
 Constructs {#constructs}
 =========================
 
 A [=service worker registration=] has an associated <dfn>list of periodicsync registrations</dfn> whose element type is a [=periodicsync registration=].
 
-A <dfn>periodicsync registration</dfn> is a tuple consisting of a [=tag=], a dictionary of options, and a [=registration state=].
+A <dfn>periodicsync registration</dfn> is a tuple consisting of a [=tag=], a dictionary of options, a <dfn>time to fire</dfn> and a [=registration state=].
 
 The <dfn>tag</dfn> is a {{DOMString}}. Within one [=list of periodicsync registrations=] each [=periodicsync registration=] MUST have a unique [=tag=]. Periodic Background Sync doesn't share namespace with Background Sync, so an origin can have registrations of both types with the same tag.
 
 The options are a dictionary containing [=minInterval=], a long long. <dfn>minInterval</dfn> is used to specify the minimum interval, in milliseconds, at which the periodic synchronization should happen. [=minInterval=] is a suggestion to the user agent. The actual interval at which <code>periodicsync</code> events are fired MUST be greater than or equal to this.
 Enclosing options in a dictionary allows this spec to be extended with more options in the future without adversely affecting existing usage.
 
+The [=time to fire=] is the soonest time decided by the user agent at which it is appropriate to fire the next <periodicsync> event for the [=periodicsync registration=].
```

Comment:
If you were creating an implementation based on this spec, what do you need to know in order to have it do something sensible? This needs more colour on _how_ a user agent can decide such a time, what considerations to make, and so on.

Reply:
done.